nextcloud_update.sh

```diff
@@ -625,6 +625,13 @@ then
     fi
 fi
 
+# Update updatenotification.sh (gets updated after each nextcloud update as well; see down below the script)
+if [ -f $SCRIPTS/updatenotification.sh ] && ! grep -q "Check for supported Nextcloud 21" "$SCRIPTS/updatenotification.sh"
```

Comment:
Shouldn't 21 be a $var?

Reply:
> Shouldn't 21 be a $var?

you  mean in lib.sh?

Reply:
"Check for supported Nextcloud 21"

With this we need to change for each version, use the function to check old versions instead and add that instead of hardcorded "21".

Reply:
done

Reply:
Nope, like this: https://github.com/nextcloud/vm/blob/master/lib.sh#L131

Already exists. ;)

Reply:
Hope that works now...